### DISCUSSION TITLE
Does quarto have multilingual support?

### DISCUSSION THREAD

**Discussion by kozo2** [quarto-dev/quarto-cli#518] [Q&A] [answered]
Hi quarto-dev.
I'm Kozo. I'm trying to create a multilingual website for the Japanese community of the Carpentries.
Do you have a plan for such a multilingual support?

**Comment by jjallaire** [comment, marked as answer]
We currently have translations of our UI strings available for a handful of languages (but currently not Japanese): https://quarto.org/docs/authoring/language.html. If you do create a Japanese translation it would be awesome if you could contribute it back as a PR.

Is that what you are referring to or are you thinking of sites that flip into different language modes? For the latter you'd probably just create distinct projects for each version of the site (possibly sym-linking shared resources). This isn't a fully baked solution but it is certainly possible.

**Comment by kozo2** [comment]
> We currently have translations of our UI strings available for a handful of languages (but currently not Japanese): https://quarto.org/docs/authoring/language.html. If you do create a Japanese translation it would be awesome if you could contribute it back as a PR.

Thank you for the information. I didn't know that.
I will send a PR to https://github.com/quarto-dev/quarto-cli/tree/main/src/resources/language in the future.

> Is that what you are referring to or are you thinking of sites that flip into different language modes? For the latter you'd probably just create distinct projects for each version of the site (possibly sym-linking shared resources). This isn't a fully baked solution but it is certainly possible.

Thank you for the information.
I intend the latter.
So I'm going to link distinct projects for each version of the site.

**Reply by eitsupi** [reply]
@kozo2 I have created a draft PullRequest #601.
I am not confident in my translation, so if you don't mind, could you please review it?

**Reply by kozo2** [reply]
@eitsupi Thank you for moving things forward!
I started the review.
Give me some time to finish it.

**Comment by maelle** [comment]
Related feature request: https://github.com/quarto-dev/quarto-cli/issues/275
